### DISCUSSION TITLE
是否考虑兼容鸿蒙

### DISCUSSION THREAD

**Discussion by zhongjiahui** [Tencent/libpag#2148] [Q&A] [answered]
请问下是否有兼容鸿蒙系统计划？是否会推出鸿蒙系统下的sdk？

**Comment by kevingpqi123** [comment, marked as answer]
有计划，现在的问题是还没有鸿蒙设备，腾讯内其它业务目前还处于评估阶段
微信、手Q、腾讯视频、腾讯会议、腾讯地图等主流业务都在使用 pag， 他们适配鸿蒙的时候同样需要 pag 提前适配好

**Reply by ldw5821cn** [reply]
适配完会开源出来吗？或者有进一步消息辛苦及时通知给我们

**Reply by kevingpqi123** [reply]
社区版的适配源码会公布出来，pag 的核心代码都是 C++ 层，C++ 层实现各平台都是共用的，这里的适配仅仅涉及平台层封装、OpenGL 接口绑定以及硬解适配，源码量不会太大
